security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/config/VertxAuthObjectMapperCustomizer.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.ts.security.vertx.config;
+
+import javax.inject.Singleton;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import io.quarkus.jackson.ObjectMapperCustomizer;
+
+@Singleton
+public class VertxAuthObjectMapperCustomizer implements ObjectMapperCustomizer {
+
+    @Override
+    public void customize(ObjectMapper objectMapper) {
+        // TODO https://github.com/quarkusio/quarkus/issues/18927
```

Comment:
This issue has been fixed and we can now do this via properties. Pls, update the PR accordingly.